ReflectionTest/ReflectionConsoleTest/Program.cs

```diff
@@ -12,62 +12,61 @@ class Program
     {
         static void Main(string[] args)
         {
-            try
+            var user = new User("tsar", "12345", true);
+            var users = new UserStorage();
+
+            for (int i = 0; i < 20; i++)
             {
-                var user = new User("tsar", "12345", true);
-                var users = new UserStorage();
+                users.AddRandomUser();
+            }
 
-                for (int i = 0; i < 20; i++)
-                {
-                    users.AddRandomUser();
-                }
+            users.AddUser(user);
+            users.DeleteUser(users.FindByLogin("tsar"));
+            users.DeleteUser(user);
 
-                users.AddUser(user);
-                users.DeleteUser(users.FindByLogin("tsar"));
-                users.DeleteUser(user);
-                users.DeleteUser(user);
+            GetReflectionInfo("SomeLibrary");
```

Comment:
Normally assemblies have a lot of different namespaces, but it's ok for now.

Reply:
Ok.